src/pybind/mgr/dashboard/frontend/src/app/core/auth/role-list/role-list.component.ts

```diff
@@ -90,13 +90,13 @@ export class RoleListComponent implements OnInit {
   }
 
   deleteRoleModal() {
-    this.modalRef = this.modalService.show(DeletionModalComponent);
     const name = this.selection.first().name;
-    this.modalRef.content.setUp({
-      metaType: 'Role',
-      pattern: `${name}`,
-      deletionMethod: () => this.deleteRole(name),
-      modalRef: this.modalRef
+    this.modalRef = this.modalService.show(DeletionModalComponent, {
+      initialState: {
+        metaType: 'Role',
```

Comment:
The `actionDescription` is missing here which results in a broken modal and that the following error is thrown in console `No action description defined`.

Reply:
Nice find